src/content/docs/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,188 @@
+
+If you have not upgraded to the latest major version of Astro, you may encounter significant differences between the Astro documentation and your project's behavior. We strongly recommend upgrading to the current major version of Astro as soon as you are able. Both the code and the documentation for earlier versions is unsupported.
+
+### Upgrade Guides
+
+After every [major version release](#major-changes), you will find an **upgrade guide** with information about important changes and instructions for upgrading your project code.
+
+The main Astro documentation pages are always **accurate for the latest released version of Astro**. They do not describe or compare to how things worked in previous versions, nor do they highlight updated or changed behavior.
+
+See the upgrade guides below for an explanation of changes, comparing the new version to the old. The upgrade guides include everything that could require you to change your own code: breaking changes, deprecations, feature removals and replacements as well as updated usage guidance. Each change to Astro includes a "What should I do?" section to help you successfully update your project code.
+
+- [Upgrade to v4](/en/guides/upgrade-to/v4/)
+- [Upgrade to v3](/en/guides/upgrade-to/v3/)
+- [Upgrade to v2](/en/guides/upgrade-to/v2/)
+- [Upgrade to v1](/en/guides/upgrade-to/v1/)
+
+### Older docs (unmaintained)
+
+Documentation for older versions of Astro is not maintained, but is available as a static snapshot. Use these versions of docs if you are unable to upgrade your project, but still wish to consult guides and reference:
+
+- [unmaintained v3.6.3 snapshot](https://docs-git-v3-docs-unmaintained-astrodotbuild.vercel.app/)
+- [unmaintained v2.10.15 snapshot](https://deploy-preview-4405--astro-docs-2.netlify.app/en/getting-started/)
+
+
+## Semantic versioning
+
+Astro attempts to adhere as much as possible to [semantic versioning](https://semver.org/), which is a set of rules developers use to determine how to assign a version number to a release. Semantic version follows predictable to inform users of the kind of changes they can expect from one version to the next.
+
+Semantic versioning enforces a pattern of `X.Y.Z` for software version numbers. These values represent **major (X)**, **minor (Y)**, and **patch (Z)** updates.
+
+### Patch changes
+
+Patch changes are the least disruptive changes. They do not change the way you use Astro, and no change to your own code is required when you update.
+
+When Astro issues a "patch" version, the last number increases. (e.g. `astro@4.3.14` -> `astro@4.3.15`)
+
+Patches may be released for reasons such as:
+
+- Re-releases after a failed release.
+- Internal changes that do not change Astro’s functionality:
+  - refactors
+  - performance improvements
+  - increase or change in test coverage
+  - aligning with stated documentation and expected behavior
+- Improvements to logging and error messages.
```

Comment:
```suggestion
- Internal changes that do not change Astro’s functionality:
  - refactors
  - performance improvements
  - increase or change in test coverage
  - aligning with stated documentation and expected behavior
- Improvements to logging and error messages.
- Re-releases after a failed release.
```

Reply:
This order made more sense to me, since the vast majority of changes are the first point.